src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ReferenceDataTest.kt

```diff
@@ -365,9 +368,63 @@ class ReferenceDataTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `Get Move on Categories for only approved premises returns 200 with correct body`() {
-    moveOnCategoryRepository.deleteAll()
+  fun `Get Move on Categories with includeInactive flag set returns active and inactive move on categories`() {
+    moveOnCategoryEntityFactory.produceAndPersistMultiple(10)
+    inactivateExistingMoveOnCategories()
+    moveOnCategoryEntityFactory.produceAndPersistMultiple(10)
+
+    val jwt = jwtAuthHelper.createValidAuthorizationCodeJwt()
+
+    val result =
+      webTestClient
+        .get()
+        .uri("/reference-data/move-on-categories?includeInactive=true")
```

Comment:
Is there a test for `includeInactive=false`?  includeInactive is nullable by the looks of it; is there a test for `includeInactive=null `too?

Reply:
Should `includeInactive` have a default value in the spec then?  I can't see it. 

It is not required so null, true or false could be supplied.

Reply:
Ahh ok, I see how this works [here](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/705b73b7c5ca9d46dcd142b150b4644b7b2d72ce/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/ReferenceDataController.kt#L101).

I think the tests for not suppling it are covered off already in your other tests.

Probably just a test to check that supplying it equal to false gives the same as supplying a null value?

Would be good I think since we're covering off all bases of this signature:

```
  override fun referenceDataDepartureReasonsGet(
    xServiceName: ServiceName?,
    includeInactive: Boolean?,
  )
```

Will approve in any case.